pages/who-we-are/index.js

```diff
@@ -26,6 +27,12 @@ const FAQDropdown = ({ prompt, content }) => {
 const WhoWeAre = () => {
   return (
     <div className="flex flex-col w-full">
```

Comment:
we probably want `-mt-5` on mobile so the image goes behind the rounded corners for the mobile navbar. Right now there is space in the corners